src/main/java/at/hannibal2/skyhanni/features/misc/FixGhostEntities.kt

```diff
@@ -55,5 +57,14 @@ object FixGhostEntities {
         }
     }
 
+    @SubscribeEvent
+    fun onCheckRender(event: CheckRenderEntityEvent<*>) {
+        if (!LorenzUtils.inSkyBlock || !config.hideTemporaryArmorstands) return
+        if (event.entity !is EntityArmorStand) return
+        with(event.entity) {
+            if (ticksExisted < 10 && name == "Armor Stand" && inventory.all { it == null }) event.cancel()
```

Comment:
ModUtils.isDefaultValue should be used to check if an armor stand has default name

Reply:
oh thats a thing, huh. will do